Controllers/ImageRecognizer.cs

```diff
@@ -31,29 +50,62 @@ public void RecognizeTestImages()
             catch (Exception e)
             {
                 Debug.WriteLine(e);
-                throw;
             }
         }
 
-        private string Recognize(FileInfo imagePath)
+        private string Recognize(FileSystemInfo imagePath)
```

Comment:
Why change this?

Reply:
Resharper suggested :D As I understand, it makes parameter more general.